Controller/Adminhtml/Exportcategory/Export.php

```diff
@@ -1,29 +1,30 @@
 <?php
 /**
-* Navin Bhudiya
-* Copyright (C) 2016 Navin Bhudiya <navindbhudiya@gmail.com>
-*
-* NOTICE OF LICENSE
-*
-* This program is free software: you can redistribute it and/or modify
-* it under the terms of the GNU General Public License as published by
-* the Free Software Foundation, either version 3 of the License, or
-* (at your option) any later version.
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-* GNU General Public License for more details.
-*
-* You should have received a copy of the GNU General Public License
-* along with this program. If not, see http://opensource.org/licenses/gpl-3.0.html.
-*
-* @category Navin
-* @package Navin_ImportExportCategory
-* @copyright Copyright (c) 2016 Mage Delight (http://www.navinbhudiya.com/)
-* @license http://opensource.org/licenses/gpl-3.0.html GNU General Public License,version 3 (GPL-3.0)
-* @author Navin Bhudiya <navindbhudiya@gmail.com>
-*/
+ * Navin Bhudiya
```

Comment:
what has been changed in this PHPDockBlock? do we need it in general ?

Reply:
@callum-atwal why has this been resolved?

Reply:
There's no change in the actual DocBlock

Reply:
![image](https://user-images.githubusercontent.com/22006850/45302789-246a9580-b50c-11e8-922c-338297af8cd7.png)

Reply:
![image](https://user-images.githubusercontent.com/22006850/45302801-2df3fd80-b50c-11e8-944e-538acbecb3f3.png)

Reply:
![image](https://user-images.githubusercontent.com/42971429/45302816-36e4cf00-b50c-11e8-88af-1d4fed7ae41f.png)

Reply:
if there is no change then why is git showing?

Reply:
Line added after DocBlock

Reply:
compare with dev branch and see what else has been changed. @pykettk might help you with that.